cmake/ClangToolchain.cmake

```diff
@@ -0,0 +1,35 @@
+set(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR arm)
```

Comment:
Should be `aarch64` no ? To be more specific to 64-bit

Reply:
See https://stackoverflow.com/questions/70475665/what-are-the-possible-values-of-cmake-system-processor
And https://stackoverflow.com/questions/45125516/possible-values-for-uname-m/45125525#45125525